python/pyspark/pandas/plot/core.py

```diff
@@ -857,13 +857,12 @@ def box(self, precision=0.01, **kwds):
 
         Parameters
         ----------
-        precision: scalar, default = 0.01
-            This argument is used by pandas-on-Spark to compute approximate statistics
-            for building a boxplot. Use *smaller* values to get more precise
-            statistics.
         **kwds : optional
             Additional keyword arguments are documented in
             :meth:`pyspark.pandas.Series.plot`.
+            One of the optional keyword arguments is `precision`, a float that is used by
```

Comment:
Let's probably follow numpydoc's kwargs style (https://numpydoc.readthedocs.io/en/latest/format.html#parameters)

```
    Additional arguments should be passed as keyword arguments
**kwargs : dict, optional
    Extra arguments to `metric`: refer to each metric documentation for a
    list of all possible arguments.

```

Reply:
Sorry I might not fully understand what "numpydoc's kwargs styple" refers to here,

```
        **kwds : dict, optional
            Additional keyword arguments are documented in
            :meth:`pyspark.pandas.Series.plot`.
        precision: scalar, default = 0.01
            This argument is used by pandas-on-Spark to compute approximate statistics
            for building a boxplot. Use smaller values to get more precise
            statistics.
```
is the above what we want?

Reply:
I actually meant something like

```
**kwds : dict, optional
    Extra arguments to `precision `: refer to a float that is used by ...
```

Reply:
I didn't realize `metric` in the example is a parameter like `precision`..
Thank you that's so clear!